lti_consumer/lti_xblock.py

```diff
@@ -304,26 +300,15 @@ class LtiConsumerXBlock(StudioEditableXBlockMixin, XBlock):
             "Basic Outcomes requests.</b>"
         ),
     )
-    # Client ID and block key
+    # DEPRECATED - These variables were moved to the LtiConfiguration Model
     lti_1p3_client_id = String(
-        display_name=_("LTI 1.3 Block Client ID"),
+        display_name=_("LTI 1.3 Block Client ID - DEPRECATED"),
```

Comment:
Can we fully delete these eventually?

Reply:
@nedbat We can delete these fields after applying migration 0005 included in this PR.